data/tutorials/gs_01_a_first_hour_with_ocaml.md

```diff
@@ -1,47 +1,57 @@
 ---
 id: first-hour
-title: A First Hour with OCaml
+title: Your First Day with OCaml
 description: >
   Discover the OCaml programming language in this longer tutorial that takes you
   from absolute beginner to someone who is able to write programs in OCaml.
 category: "getting-started"
 date: 2021-08-06T17:11:00-00:00
 ---
 
-# A First Hour with OCaml
+# Your First Day with OCaml
 
-You may follow along with this tutorial with just a basic OCaml installation,
+This tutorial will not only give you the basics of OCaml, but it will also guide you through creating your first applications with OCaml. You can follow along with a basic OCaml installation,
 as described in [Up and Running](/docs/up-and-running).
 
 Alternatively, you may follow almost all of it by running OCaml in your browser
 using [TryOCaml](http://try.ocamlpro.com), with no installation required.
 
-On macOS/iOS/iPadOS, you can download this [all-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side-by-side with an interactive top level, and is free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS).
+On macOS/iOS/iPadOS, you can download this [all-in-one package on the App Store](https://apps.apple.com/app/ocaml-learn-code/id1547506826). It contains an editor side-by-side with an interactive toplevel. Plus, it's free and [open source](https://github.com/GroupeMINASTE/OCaml-iOS).
+
+**Please note**: this tutorial assumes you have some programming experience and are familiar with basic concepts like functions, variables, loops, arrays, etc. 
+
 
 ## Running OCaml Programs
 
-To try small OCaml expressions, use an interactive toplevel, or REPL
-(Read-Eval-Print Loop). The `ocaml` command provides a basic toplevel
-(install `rlwrap` through your system package manager and run `rlwrap ocaml` instead to get history navigation).
+We will be working with small OCaml expressions at first, so using an interactive toplevel, or REPL
+(Read-Eval-Print Loop), will suffice.
+
+We recommend using UTop (see [here](/docs/up-and-running#using-the-ocaml-toplevel-with-utop)),
+but alternatively, the basic toplevel of OCaml (`ocaml` command) can also be used. For installation instructions,
+see [Up and Running](/docs/up-and-running#installing-ocaml). 
+
+If you have never used a toplevel (REPL) before, think of it as an interactive terminal/shell that
+evaluates expressions. You type an expression followed by the Enter or Return key,
+and then the toplevel responds with the value of the evaluated expression. 
```

Comment:
```suggestion
and then the toplevel responds with the calculated value.
```
Would this wording work? Just to avoid "evaulated expression" since we just said it evaluates expressions and again in the next paragraph.

Reply:
That's perfectly fine. Sometimes I overdo it with the precise language and since the tutorial already mentions the keyword "evalute" a moment ago, using a different way to say the same thing makes it easier to understand.